anonympy/pdf/core_pdf.py

```diff
@@ -568,19 +555,16 @@ def _find_EOI(self,
         >>> names = anonym._find_EOI(text, "PER")
         """
         coords = {}
-        page_number = 1
-
-        for excerpt in text:
+        for page_number, excerpt in enumerate(text, 1):
             ner = self._nlp(excerpt)
             names = []
             bbox = []
 
             find_EOI(ner, names, EOI)
             find_coordinates_pytesseract(names,
-                                         self.pages_data[page_number-1],
+                                         self.pages_data[page_number - 1],
                                          bbox)
             coords[f'page_{page_number}'] = bbox
-            page_number += 1
         return coords
 
     def find_ORG(self, text: List[str]) -> Dict[str, List[Tuple[int]]]:
```

Comment:
Function returns None

Reply:
`find_ORG` looks for organization names and `None` means none found in the given input